activemq-jms-pool/src/main/java/org/apache/activemq/jms/pool/PooledConnectionFactory.java

```diff
@@ -271,7 +272,39 @@ public synchronized Connection createConnection(String userName, String password
         return newPooledConnection(connection);
     }
 
-    protected Connection newPooledConnection(ConnectionPool connection) {
+    /**
+     * @return Returns the JMSContext.
+     */
+    @Override
+	public JMSContext createContext() {
+    	throw new UnsupportedOperationException("createContext() is not supported");
+	}
+
+    /**
+     * @return Returns the JMSContext.
+     */
+	@Override
+	public JMSContext createContext(String userName, String password) {
+		throw new UnsupportedOperationException("createContext() is not supported");
```

Comment:
I like the way you're calling out the specific methods in the exceptions. Makes it clear to diagnose. But you should standardize on having `"createContext(String, String)"` or just `"createContext()"`, and same with the other unimplemented methods.

Reply:
Just missed this one.. I'll fix in the follow-up commit this morning